### PR TITLE
Add simple table drawing functionality

### PR DESCRIPTION
**Pull request by martinpfannemueller**

### CODE REVIEW

**Comment by nRewik** [comment]
Thanks. Looking good for create a simple table with text. 👍

File: SimplePDF/SimplePDF.swift [hunk 1]
```diff
@@ -244,6 +249,86 @@ public class SimplePDF{
...
+        let paraStyle = NSMutableParagraphStyle()
+        
+        let skew = 0.0
+        
+        let attributes: NSDictionary = [
+            NSForegroundColorAttributeName: fieldColor,
+            NSParagraphStyleAttributeName: paraStyle,
+            NSObliquenessAttributeName: skew,
+            NSFontAttributeName: fieldFont!
+        ]
```
Comment (nRewik) [inline SimplePDF/SimplePDF.swift]:
```
let attributes: [String: AnyObject] = [
		NSForegroundColorAttributeName: fieldColor,
		NSParagraphStyleAttributeName: paraStyle,
		NSObliquenessAttributeName: skew,
		NSFontAttributeName: fieldFont!
	]
```

File: SimplePDF/SimplePDF.swift [hunk 2]
```diff
@@ -244,6 +249,86 @@ public class SimplePDF{
...
+        
+        let size = text.sizeWithAttributes(attributes as? [String : AnyObject]);
+        
+        let x = (rect.size.width - size.width)/2
+        let y = (rect.size.height - size.height)/2
+        
+        
+        text.drawAtPoint(CGPointMake(rect.origin.x + x, rect.origin.y + y), withAttributes: attributes as? [String : AnyObject])
```
Comment (nRewik) [inline SimplePDF/SimplePDF.swift]: remove ` as? [String : AnyObject]`

**Comment by nRewik** [comment]
`;` should not appear in Swift. Remove all semicolons and it should be ok.

File: SimplePDF/SimplePDF.swift [hunk 3]
```diff
@@ -244,6 +249,86 @@ public class SimplePDF{
...
+        CGContextSetStrokeColorWithColor(context, color);
+        CGContextMoveToPoint(context, from.x, from.y);
+        CGContextAddLineToPoint(context, to.x, to.y);
+        
+        CGContextStrokePath(context);
+    }
+    
+    private func drawTextInCell(rect: CGRect, text: NSString)
+    {
+        let fieldColor: UIColor = UIColor.blackColor()
```
Comment (nRewik) [inline SimplePDF/SimplePDF.swift]:
```
let fieldColor = UIColor.blackColor()
```

**Comment by martinpfannemueller** [comment]
Thanks for your feedback! I added an option for specifying the font size and cleaned everything up.

**Comment by nRewik** [comment]
One more thing.  Now the font is fixed to `Helvetica Neue`. We should let user to specify both type and size of font to show in the table.

So, `addTable` function might be like this.

```
addTable(
    rowCount rowCount: Int, 
    columnCount: Int, 
    rowHeight: CGFloat, 
    columnWidth: CGFloat, 
    tableLineWidth: CGFloat, 
    font: UIFont, 
    dataArray: Array<[String]>
)
```

And it can be used like this, which I think it is nicer.
```
addTable(
    rowCount: 1, 
    columnCount: 1, 
    rowHeight: 20, 
    columnWidth: 20, 
    tableLineWidth: 1, 
    font: UIFont.systemFontOfSize(UIFontWeightRegular), 
    dataArray: dataArray
)
```

**Comment by nRewik** [comment]
Great work. 👍

**Pull request merged**